pypeit/spectrographs/keck_lris.py

```diff
@@ -346,6 +398,44 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
 
         msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
         return np.zeros(len(fitstbl), dtype=bool)
+
+    def vet_assigned_ftypes(self, type_bits, fitstbl):
+        """
+        Check the assigned frame types for consistency.
+        Args:
+            type_bits (`numpy.ndarray`_):
+                Array with the frame types assigned to each frame
+            fitstbl (`PypeItMetaData`_):
+                Table with the metadata for the frames to check.
+
+        Returns:
+            `numpy.ndarray`_: The updated frame types.
+
+        """
+        type_bits = super().vet_assigned_ftypes(type_bits, fitstbl)
+
+        # If both pixelflat and slitless_pixflat are assigned in the same configuration, remove pixelflat
+
+        # where slitless_pixflat is assigned
+        slitless_idx = fitstbl.type_bitmask.flagged(type_bits, flag='slitless_pixflat')
+        # where pixelflat is assigned
+        pixelflat_idx = fitstbl.type_bitmask.flagged(type_bits, flag='pixelflat')
```

Comment:
Use `find_frames` instead?

Reply:
This method is actually used before the `frametype` is created, this is why I am using bits here.